Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,181 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
+
+2. What is the population of Rotterdam ?
+
+select Population from city where city.Name="Rotterdam";
++------------+
+| Population |
++------------+
+|     593321 |
++------------+
+1 row in set (0.00 sec)
+
+3. What's the name of all the countries on the continent ‘North America’ ?
+
+ select Name from country where country.Continent="North America";
++----------------------------------+
+| Name                             |
++----------------------------------+
+| Aruba                            |
+| Anguilla                         |
+| Netherlands Antilles             |
+| Antigua and Barbuda              |
+| Bahamas                          |
+| Belize                           |
+| Bermuda                          |
+| Barbados                         |
+| Canada                           |
+| Costa Rica                       |
+| Cuba                             |
+| Cayman Islands                   |
+| Dominica                         |
+| Dominican Republic               |
+| Guadeloupe                       |
+| Grenada                          |
+| Greenland                        |
+| Guatemala                        |
+| Honduras                         |
+| Haiti                            |
+| Jamaica                          |
+| Saint Kitts and Nevis            |
+| Saint Lucia                      |
+| Mexico                           |
+| Montserrat                       |
+| Martinique                       |
+| Nicaragua                        |
+| Panama                           |
+| Puerto Rico                      |
+| El Salvador                      |
+| Saint Pierre and Miquelon        |
+| Turks and Caicos Islands         |
+| Trinidad and Tobago              |
+| United States                    |
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+37 rows in set (0.00 sec)
+
+4. What's the top 10 countries by SurfaceArea ?
+select Name from country where country.SurfaceArea limit 0, 10;
```

Comment:
Not quite - you needed to sort them in order by surface area.

`WHERE country.SurfaceArea` is similar to saying `if ( surfaceArea )` - but in mysql, it only checks that it's not null.

Reply:
Hi @mahmutkaya or @DianaAyoub , did you fix or comment here yet?

Reply:
not yet :)

Reply:
it's done. but I couldn't push to here, so I did new pullR.